axelor-purchase/src/main/java/com/axelor/apps/purchase/service/PurchaseOrderLineService.java

```diff
@@ -104,9 +104,6 @@ public void checkMultipleQty(
       ActionResponse response)
       throws AxelorException;
 
-  public String[] getProductSupplierInfos(
```

Comment:
Is there a reason for removing the public method ? I can't see a direct link with the origin of the anomaly (https://github.com/axelor/axelor-open-suite/pull/10161/files)  

Even if it is not used anymore in dev, might be the case in previous versions so it should be deleted in futur minor versions. If the method need to be deleted in all cases and another method can be provided, please add developer section in the changelog to notify the devs of this change and the substitute method

Reply:
Separate task is created in Redmine to remove this unused method: https://redmine.axelor.com/issues/79212